pom.xml

```diff
@@ -204,5 +210,24 @@
 			</properties>
 		</profile>
 
+		<profile>
```

Comment:
It would be great if we could get this as a different PR.

Reply:
If we remove this profile section, project build for Java 11 and Java 17 will fail for this PR

Reply:
We could rebase that commit to this pull request. So that won't be a problem.

Reply:
@ManojLL I merged the pull request for Java 17 support.

Reply:
I debase the branch with java 17 changes